src/zenml/logger.py

```diff
@@ -193,9 +340,27 @@ def init_logging() -> None:
     os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
     set_root_verbosity()
 
-    console_handler = logging.StreamHandler(sys.stdout)
-    console_handler.setFormatter(get_formatter())
-    logging.root.addHandler(console_handler)
+    # Check if console handler already exists to avoid duplicates
+    root_logger = logging.getLogger()
+    has_console_handler = any(
+        isinstance(handler, logging.StreamHandler)
+        and handler.stream == sys.stdout
+        for handler in root_logger.handlers
+    )
+
+    if not has_console_handler:
+        console_handler = logging.StreamHandler(sys.stdout)
+        console_handler.setFormatter(get_formatter())
+        # Set console handler level explicitly to console verbosity
+        console_handler.setLevel(get_logging_level().value)
+        root_logger.addHandler(console_handler)
+
+    # Initialize global print wrapping
+    try:
+        setup_global_print_wrapping()
+    except ImportError:
```

Comment:
Why would there be an import error here?

Reply:
Removed the try-except.